xml/docker_dockerizing_application.xml

```diff
@@ -314,81 +314,161 @@ ENTRYPOINT ["apache2ctl"]</screen>
+    <callout arearefs="co.dockerfile.apache2.copy">
+     <para>The copy instruction to copy your own data to the
+      server's directory. Currently, this line is disabled due to
+      the hash mark in the first column.
+     </para>
+    </callout>
+    <callout arearefs="co.dockerfile.apache2.expose">
+     <para>The exposed port for the Apache Web server.</para>
+    </callout>
+   </calloutlist>
+  </example>
 
-VOLUME ["/var/spool/mail"]
-ENTRYPOINT ["/usr/sbin/postfix"]</screen>
-
-  <para>
-   Now you can build your image. &deng; performs the following operations
-   according to the instructions in the <filename>Dockerfile</filename>:
-  </para>
+  <note>
+   <title>Check for Running Apache2 Instances on Your Host</title>
+   <para>Make sure that you do not have any Apache2 server instances
+    running on your host. The &docker; container would not serve any data if you
+    have a host Apache2 server running. Remove or stop any Apache2 servers on
+    your host. </para>
+  </note>
 
-  <orderedlist>
-   <listitem>
-    <para>
-     &deng; refreshes repositories and installs the
-     <emphasis>postfix</emphasis> mail server as it is not installed by default
-     in the &slsa; &docker; image.
-    </para>
-   </listitem>
-   <listitem>
-    <para>
-     The <literal>/var/spool/mailboxes</literal> directory is created in the
-     file system of the image. The directory will store all mailboxes if you
-     configure the mail server to store data in this directory.
+  <para>To use the container, proceed as follows:</para>
+  <procedure xml:id="pro.docker.apache2">
+   <title>Testing the Apache2 Web Server</title>
+   <step>
+    <para>Prepare your system for the build process:</para>
+    <substeps>
+     <step>
+      <para>Make sure your host is subscribed to the <guimenu>Server Applications Module</guimenu>.
+       Open &yast; and select <guimenu>Add System Extensions or Modules</guimenu>
+       to see the available modules and extensions.</para>
+     </step>
+     <step>
+      <para>Make sure you have installed the &sle; images from &suse; Registry
+       as described in <xref linkend="Building_Pre-build_Images"/>.</para>
+     </step>
+     <step>
+      <para>Save the <filename>Dockerfile</filename> from <xref
+       linkend="ex.docker.dockerfile.apache2"/> into the
+       <filename class="directory">docker</filename> directory.</para>
+     </step>
+     <step>
+      <para>Copy the registered repositories and service files to the
+      <filename class="directory">docker/etc</filename> directory:</para>
+      <screen>cd docker
+mkdir etc
+sudo cp -a /etc/zypp/{repos.d,services.d} etc/</screen>
+      <para>This step is needed to expose the registered repositories
+       (<systemitem>SLE-Module-Server-Applications15-Pool</systemitem>) and
+       services on your host for your &docker; container.
+       If wanted, it is possible to copy only the needed repository or service
+       files to the &docker; container.
```

Comment:
Instead of copying all repository and service files, you can copy only the ones that are required by the &docker; container.